firebase-perf/src/main/java/com/google/firebase/perf/logging/ConsoleUrlGenerator.java

```diff
@@ -52,8 +52,19 @@ public static String generateCustomTraceUrl(String projectId, String packageName
    * @param name the trace name
    */
   public static String generateScreenTraceUrl(String projectId, String packageName, String name) {
+    String rootUrl = getRootUrl(projectId, packageName);
+    return String.format(
+        "%s/metrics/trace/SCREEN_TRACE/%s?utm_source=%s&utm_medium=%s",
+        rootUrl, name, UTM_SOURCE, UTM_MEDIUM);
+  }
+  /**
```

Comment:
Nit: Leave an empty line between the end of a function and the javadoc of the next function :)

Reply:
Done. Thanks for the speedy feedback!